docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -69,7 +73,7 @@ OpenTelemetry Collectors can be run as an Agent, or as standalone collectors.
 They can be deployed as often as necessary and scaled up or out.
 
 Deployment planning resources are available in OpenTelemetry's {ot-collector}[Getting Started]
```

Comment:
This link is also a 404 and should be updated

Reply:
Thanks. Added https://opentelemetry.io/docs/collector/getting-started/